documentation/List-of-Action-plugins.md

```diff
@@ -175,10 +176,12 @@ When a reviewer self-assigns themselves to a review_assignment (i.e. its status
 
 - _Action Code:_ **`updateReviewAssignmentsStatus`**
 
-| Input parameters<br />(\*required) <br/>            | Output properties                                     |
-| --------------------------------------------------- | ----------------------------------------------------- |
-| `reviewAssignmentId`\*                              | `reviewAssignmentUpdates` (`array` of `{id, status}`) |
-| `trigger`\* (only executes on `onReviewSelfAssign`) |                                                       |
+| Input parameters<br />(\*required) <br/>          | Output properties                                     |
+| ------------------------------------------------- | ----------------------------------------------------- |
+| `reviewAssignmentId`\*                            | `reviewAssignmentUpdates` (`array` of `{id, status}`) |
+| `trigger` (only executes on `onReviewSelfAssign`) |                                                       |
```

Comment:
It just means that if the trigger is not `onReviewSelfAssign`, the plugin won't do anything. (Most other plugins can be triggered from any trigger, but to do so for any other trigger on this one would fail, since it needs a reviewAssignmentId record as input)